src/plots/gl3d/scene.js

```diff
@@ -70,6 +71,7 @@ function render(scene) {
     if(lastPicked !== null) {
         var pdata = project(scene.glplot.cameraParams, selection.dataCoordinate);
         trace = lastPicked.data;
+        var traceNow = gd._fullData[trace.index];
```

Comment:
@etpinard Could you please describe why `traceNow` is needed here?
Thanks in advance.

Reply:
Because `hoverlabel.*` edits don't (and shouldn't have to) replot the scene. As `Plots.supplyDefaults` creates new `gd._fullData` and `gd._fullLayout` refs on every call, we need to use the right "now" trace object when hovering.